### PR TITLE
Fixes for JupyterLab 1.0.0

### PR DESCRIPTION
**Pull request by agoose77**

### CODE REVIEW

**Comment by agoose77** [comment]
Not sure if there were any modified icons by this theme - I just took them from master in theme-dark-extension

**Comment by agoose77** [comment]
I didn't restore `scrollbars.css` as I felt this might have been ommitted for a reason. I just returned the basic scrollbar theming.

**Comment by telamonian** [comment]
Thanks for helping me get this up to date!

Don't need `scrollbars.css`, since the contents of that were already pushed upstream to jlab itself.

I looked over your changes, and the only thing that caught my eye is that there's probably still a bunch of now-unused icons sitting around (if I'm interpreting the changes you made to ` style/urls.css` correctly). There's no custom icons in this theme, so you can just nuke `style/icons` and `style/images` and replace them with the latest versions from `@jupyterlab/theme-dark-extension`.

Other than that, I think it looks ready to pull. I also need to sort out why the CI is failing, but that probably has nothing to do with this PR.

**Comment by agoose77** [comment]
Hey no problem, it's a great extension and it was about time I contributed more than GitHub Issues.

Yeah, if you've just exported the JLab icons, I'll overwrite them from master and update urls respectively.

**Comment by agoose77** [comment]
Updated images.

File: style/variables.css [hunk 1]
```diff
@@ -179,9 +179,9 @@ all of MD as it is not optimized for dense, information rich UIs.
    * Code font variables are used for typography of code and other monospaces content.
    */
 
-  --jp-code-font-size: 15px;
-  --jp-code-line-height: 1.3077; /* 17px for 15px base */
-  --jp-code-padding: 0.385em; /* 5px for 15px base */
+  --jp-code-font-size: 14px;
+  --jp-code-line-height: 1.214; /* 17px for 15px base */
+  --jp-code-padding: 0.357em; /* 5px for 15px base */
```
Comment (telamonian) [inline style/variables.css]: We discussed this. I already told you I'm not going to change the code font size. As an olive branch, I started working on jupyterlab/jupyterlab#6926, which will let you set your own font sizes.

Comment (agoose77) [reply style/variables.css]: I apologise Max, I pushed my local changes from my develop branch into this PR branch. I am glad to hear you'll add a mechanism to change this, though, then I can drop my branch entirely! I'll revert the commit, thanks for the input.

Comment (telamonian) [reply style/variables.css]: That's okay. The other day, for lack of a missing colon I accidentally force pushed an empty branch to someone else's long-running PR. Sorry to have been snippy.

**Comment by telamonian** [comment]
Okay, everything looks great, and I tested and can confirm it works with the latest Jlab release (1.0.4). Thanks for all your work, @agoose77!

**Comment by telamonian** [comment]
I'll pull this and then do an `npm` release

**Comment by telamonian** [comment]
Hmm, I guess I still have to rebase and/or merge with master (they've drifted apart). But I think I'll save that for a future release.

**Comment by agoose77** [comment]
No worries, I'd have felt them same if someone had snuck in an explicitly
rejected change.
Thanks for the help!

**Pull request merged**